src/main/scala/lms/transformation/distributed_tensor/distributedTensorConv.scala

```diff
@@ -86,6 +102,8 @@ trait FixedSizeDistributedTensorConvTypeLess extends FixedSizeDistributedTensorM
       val filterCin   = filter_type(CUDNN_C_IN).dim
       List((inputC, filterCin), (outputC, filterCout))
 
+    case Node(s, op, _, _) if op == "tensor_softmax" => List()
```

Comment:
would be simpler to just do `case Node(s, "tensor_softmax", _, _) => List()`

Reply:
fixed in 232d098